cpp/install_email_processor.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefil
+
+if [ -e /home/email_processor/.fetchmailrc ]; then
+    echo 'It looks as if the email_processor account has already been set up!'
+    exit 0
+fi
+
+if [[ $(whoami) != "root" ]]; then
+    echo 'You must run '"$0"' as root!'
+    exit 1
+fi
+
+if [ ! $(which fetchmail) ]; then
+    echo 'You must first install fetchmail before running this script!'
+    exit 2
+fi
+
+adduser email_processor email_processor
```

Comment:
careful! `adduser` is a special program in Ubuntu, but in CentOS it's an alias for `useradd` which exists on both systems but has different syntax.

Reply:
... and maybe test if user exists before adding it?